db/rocksdb_ethereumtype.go

```diff
@@ -53,19 +53,58 @@ func packAddrContracts(acs *AddrContracts) []byte {
 			l = packBigint(&ac.Value, varBuf)
 			buf = append(buf, varBuf[:l]...)
 		} else if ac.Type == bchain.NonFungibleToken {
-			l = packVaruint(uint(len(ac.Ids)), varBuf)
+			i := 0
+			ids := []*big.Int{}
+			ac.Ids.Range(func(key, val any) bool {
+				k, ok := key.(string)
+				if !ok {
+					glog.Errorf("invalid nft id: %v (type: %T)", key, k)
```

Comment:
to reiterate, since we know the types we are dealing with throughout the pipeline, we could shortcut the error handling to make the code less verbose, but figured the safety net to catch unintended later changes might be beneficial. easy to get lost in this area of things (at least for me haha)